web/src/engine/keyboard/src/keyboards/jsKeyboard.ts

```diff
@@ -52,7 +52,7 @@ type KmwKeyboardObject = KeyboardObject & {
  * and keyboard-centered functionality in an object-oriented way without modifying the
  * wrapped keyboard itself.
  */
-export default class Keyboard {
+export class JSKeyboard {
```

Comment:
It would be good for this and `KMXKeyboard` to inherit from an abstract base class.